packages/ember/typescript/src/eslint.js

```diff
@@ -11,4 +12,9 @@ export async function setupESLint() {
   await packageJson.addDevDependencies(depsWithVersions);
 
   // TODO: append an overrides entry
+  console.warn(
+    chalk.yellow(
+      `Updating the .eslintrc.js file to support TypeScript hasn't been automated yet.`,
```

Comment:
Are there any docs I can link here? I only found the addon / app blueprint in the ember-cli repo, but that doesn't contain a ready to copy file either.

Reply:
yeah, it's a bit tricky because:
1. I don't agree with the current `.eslintrc.js` in the ember blueprint
2. (because), it doesn't subscribe to an _overrides-only_ approach, which makes for easy dealings with parsers, etc (and this will be the _only_ options in ESLint 9)
3. we could _try_ to push in a config to handle typescript, and maybe it'd be fine? 
4. 3 is hard because pre-flat-config, ESLint has 10 or so config file formats.